models/migrations/v224.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
```

Comment:
Could you move this migration into a versioned directory?
Also, could you update the copyright heading to match the new format?